CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Changes to Calva.
 
 - [Clojure Notebooks](https://github.com/BetterThanTomorrow/calva/issues/1824)
 - Fix: [Wrong 'when condition' for command 'paredit.togglemode'](https://github.com/BetterThanTomorrow/calva/issues/1804)
+- [Color customization for inline result](https://github.com/BetterThanTomorrow/calva/issues/1831)
```

Comment:
This should go under Unreleased.

Reply:
Oh… my bad, didn't notice 2022-08-18 release

Reply:
Done ✅

Reply:
Awesome. I think the release happened while you where merging, so not easy to detect. 😄